motors/tacho_motor_class.c

```diff
@@ -255,7 +251,7 @@ struct tacho_motor_state_item {
 
 static struct tacho_motor_value_names tacho_motor_states[NUM_TM_STATES] = {
 	[TM_STATE_RUNNING]	= { "running" },
-	[TM_STATE_RAMPING]	= { "ramping" },
+	[TM_STATE_STOPPED]	= { "stopped" },
```

Comment:
I'll look at putting `TM_STATE_RAMPING` back in and removing `TM_STATE_STOPPED` - you're right.